src/reporters/json.ts

```diff
@@ -415,42 +415,48 @@ export default class JSONReporter extends BaseReporter {
         metrics,
         status,
         error,
-        ssblocks,
+        options,
       }) => {
-        await gatherScreenshots(
-          join(CACHE_PATH, 'screenshots'),
-          async (step, data) => {
-            if (ssblocks) {
-              const { blob_mime, blocks, reference } =
-                await getScreenshotBlocks(Buffer.from(data, 'base64'));
-              for (let i = 0; i < blocks.length; i++) {
-                const block = blocks[i];
+        const { ssblocks, screenshots } = options;
```

Comment:
Could we break this up as well? It's an unruly size. I can picture this being two functions, one for ssblocks, one for regular screenshots. Or, at a minimum, just extracting the ssblocks logic to its own function.

Reply:
Sure thing 👍🏽